src/main/javascript/software.bytepushers.pick3.lottery.web.TexasPick3WebScraper.js

```diff
@@ -32,7 +32,11 @@ function TexasPick3WebScraper(TxPick3WebScraperConfig) {
     };
 
     this.findNightWinningNumber = function (drawingDate) {
```

Comment:
Notice how findNightWinningNumber(), findEveningWinningNumber(), findDayWinningNumber(), and findMorningWinningNumber() methods have the same logic?  Let re-factor all these method to call a general method that does the same thing that each of these methods do.  For example, maybe this new general method could be called: scrapeWinningNumber() or scrapeForWinningNumber() or something else; but you get the point!  :)

Reply:
Refactored